tests/org.jboss.tools.livereload.test/src/org/jboss/tools/livereload/internal/server/jetty/ContentTypeAssert.java

```diff
@@ -0,0 +1,25 @@
+package org.jboss.tools.livereload.internal.server.jetty;
+
+import org.assertj.core.api.AbstractAssert;
+
+public class ContentTypeAssert extends AbstractAssert<ContentTypeAssert, String>{
+
+	protected ContentTypeAssert(String actual, Class<?> selfType) {
+		super(actual, selfType);
+	}
+
+	public ContentTypeAssert(String actual) {
+		super(actual, ContentTypeAssert.class);
+	}
+	
+	public static ContentTypeAssert assertThat(String actual) {
+	    return new ContentTypeAssert(actual);
+	  }
+	
+	public ContentTypeAssert isEqualToIgnoreCaseAndWhitespaces(String contentType) {
```

Comment:
I would just override the isEqualTo method and add a bit of explanation in the Javadoc.